cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java

```diff
@@ -154,6 +166,13 @@ public static HiveSupport checkHiveSupport(@Nullable ClassLoader hiveClassLoader
       }
     }
 
+    boolean useLatestVersionForUnknown = cConf.get(Constants.Explore.HIVE_AUTO_LATEST_VERSION).equals(
+      cConf.get(Constants.Explore.HIVE_VERSION_FOR_UNKNOWN_VERSION));
+    if (useLatestVersionForUnknown && CDH_PATTERN.matcher(hadoopVersion).matches()) {
+      LOG.info("CDH Hadoop version '{}' is unknown. Using the latest Hive version available in CDAP. ", hadoopVersion);
```

Comment:
what does it mean to talk about Hive version? To log what's the latest Hive version to be used? Is there a programatic way to get this?